lib/index.js

```diff
@@ -1082,7 +1075,7 @@ class TextMetrics{
   }
 }
 
-const loadImage = (src, options = null) => Object.assign(new Image(options), {src}).decode()
+const loadImage = (src, width, height) => Object.assign(new Image(width, height), {src}).decode()
```

Comment:
Same here for future use, rather limiting. Though I suppose it could just pass on `...arguments` which would be most flexible.

Reply:
As above, let's consider the syntax 'not set in stone' but for now maybe let's see if there's a way to have at least the `options.type="svg"` flag be derived from the data rather than an explicit arg?